templates/common/packages/utils/serviceUtils/helpers.js

```diff
@@ -14,3 +14,9 @@ export const getDataServiceUrl = (key, isMock = false) => {
   const url = endpoints.apiGee[key];
   return isMock ? `/static/mock/${url}.json` : `${apiGeeDomain}${url}`;
 };
+
+export const getApiEndPoints = (queryId, isMock = false, isGraphQlApi=false) => {

```

Comment:
This is good but need little bit of adjustment based on above comments. 

It will be best if this code follows TypeScript.

Reply:
code change done